org.jacoco.report/src/org/jacoco/report/internal/html/page/ReportPage.java

```diff
@@ -141,7 +143,7 @@ private static void breadcrumbParent(final ReportPage page,
 		if (page != null) {
 			breadcrumbParent(page.parent, div, base);
 			div.a(page, base);
-			div.text(" > ");
+			div.text(BREADCRUMP_SEPARATOR);
```

Comment:
IMO and SonarSource designer @thibaudeberwein agree with me that previously was better:

<img width="1106" alt="report" src="https://user-images.githubusercontent.com/138671/47360873-d59a4900-d6d0-11e8-87b7-f1ed79d52bce.png">